chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -912,6 +912,9 @@ def cleanup_zfs_pools(self, test_servers, action, zpool_datasets, devices_must_e
         # Ensure agents stopped to avoid interference with pool imports/exports
         self.remote_operations.stop_agents([server['fqdn'] for server in test_servers])
 
+        # Attempt to unmount all lustre targets otherwise we won't be able to export parent pool
+        [self.remote_operations.unmount_lustre_targets(server) for server in test_servers]
```

Comment:
This kind of unorderly unmounting of lustre targets (i.e. as opposed to a proper filesystem stop such as `ApiTestCaseWithTestReset.stop_filesystem()`) is what has led to #246.

Additionally, unmounting targets from underneath Pacemaker is particularly bad.  Is there any guarantee that Pacemaker is not running before we do this `unmount_lustre_targets`?

Reply:
yes, it's after clear_ha

Reply:
But `clear_ha()` does it's own [`unmount_lustre_targets()`](https://github.com/intel-hpdd/intel-manager-for-lustre/blob/pool-unimportable/chroma-manager/tests/integration/core/remote_operations.py#L1232).  Maybe this is related to #246 and now that that is merged into this branch, you can remove this?